packages/eslint-plugin/src/rules/no-unsafe-assignment.ts

```diff
@@ -37,17 +38,18 @@ export default createRule({
       requiresTypeChecking: true,
     },
     messages: {
-      anyAssignment: 'Unsafe assignment of an `any` value.',
+      anyAssignment: 'Unsafe assignment of an {{sender}} typed value.',
```

Comment:
> `{{ sender }} typed`value.

There's no need to add the _"typed"_ for _all_ error reports. Just the new one. Existing error reports don't need to say `... an any typed value`.

```suggestion
      anyAssignment: 'Unsafe assignment of an {{sender}} value.',
```

Suggestion: move the "typed" into the `data` passed by the `createDataFromSenderType` function?

Reply:
Done.